pom.xml

```diff
@@ -54,7 +54,7 @@
     </properties>
 
     <prerequisites>
-        <maven>${mavenVersion}</maven>
+        <maven>${maven.version}</maven>
```

Comment:
We should use the property name from ASF parent.

Reply:
There is no `mavenVersion` property in version 24 of the Apache Parent POM.